README.md

```diff
@@ -27,10 +27,26 @@ tracker](https://github.com/share/sharedb/issues).
 - Projections to select desired fields from documents and operations
 - Middleware for implementing access control and custom extensions
 - Ideal for use in browsers or on the server
-- Reconnection of document and query subscriptions
 - Offline change syncing upon reconnection
```

Comment:
Actually I retract my comment, this actually happens too.

[Ops are added to pending 'queue'/array](https://github.com/share/sharedb/blob/master/lib/client/doc.js#L678) and [flushed/synced on socket reconnection](https://github.com/share/sharedb/blob/master/lib/client/doc.js#L329).

This line is also correct.